components/universal-widget/src/UniversalWidget.jsx

```diff
@@ -193,6 +197,10 @@ export default class UniversalWidget extends Widget {
     getRenderer() {
         if(this.state.config) {
             const RendererComponent = renderers[this.getRendererType()];
+            if (this.state.config.x === "Time" && this.state.granularity !== null) {
```

Comment:
Use single quotes instead double, and all other places.

Reply:
Fixed in 8b2c281